include/BoardData.hxx

```diff
@@ -47,10 +47,20 @@ struct DynaBufferBus : BidirMutexes {
 struct UartBus : DynaBufferBus {};
 struct I2cBus : FixBufferBus<32u> {};
 
+typedef struct  {
+    uint8_t nbr:6;
+    uint8_t isActive:1;
+} ServoPin;
+
+typedef struct {
+    ServoPin Pin;
+    volatile unsigned int angle;
```

Comment:
This value will be changed by a backend driver from another thread; make it atomic